rfc7991bis.rnc

```diff
@@ -28,7 +28,7 @@ rfc =
    attribute iprExtract { xsd:IDREF }?,
    [ a:defaultValue = "IETF" ]
    attribute submissionType {
-     "IETF" | "IAB" | "IRTF" | "independent"
+     "IETF" | "IAB" | "IRTF" | "independent" | "editorial"
    }?,
```

Comment:
The `stream` element in line l1077 needs to have `editorial` as an option.

Reply:
See `v3.rnc` changes in https://github.com/ietf-tools/xml2rfc/pull/958/files#diff-877886f739c9ef574984f93a4885203a458d76712d3505fe5011afaf711d8126

Reply:
Added